src/services/CalculatorProcess.js

```diff
@@ -1,51 +1,52 @@
-let memory = [];
-let result = "";
-function calculatorProcess(content,visor) {
-	
-	const numbersOperators = () => String(eval((result)));
-	
-	memory += content;
+let memory = "";
+let sLength;
```

Comment:
what mean `s`? `string`? `sum`? it's not clear

Reply:
i was thinking a bout it, because it represents "Signal" it has a comment near by it.
i placed "S", because a need some char or something simple to represent the "+/-", i can do with "+/-". but i'd like to be something simpler like S(sign), E(exposure) or CS(change sign).

Reply:
for this reason we use `constant` to have this `types` more described

Reply:
ooh i get it, yea make sense to have those const that are like: `do_somenthing_here`

Reply:
done